### ISSUE TITLE
about file:  user_list_backend.php

### ISSUE THREAD

**Issue by Phil353556** [Piwigo/Piwigo#1873]
Hi,

- Update to 13.6. : done
- The file user_list_backend.php is still there even if it is an empty file
Why the file is not deleted?
What is the reason?

```
 find . -name user_list_backend.php -ls
  1177265      0 -rwxr-xr-x   1 www-data www-data        0 Feb 24 10:54 ./admin/user_list_backend.php
```

In fact I found a lot of files with zero size:
```
# find . -size 0 -ls
  1177832      0 -rwxr-xr-x   1 www-data www-data        0 Feb 24 11:42 ./themes/smartpocket/template/menubar_related_categories.tpl
  1185164      0 -rw-r--r--   1 www-data www-data        0 Feb 20 14:07 ./themes/bootstrap_darkroom/node_modules/typeface-roboto/.npmignore
  1185152      0 -rw-r--r--   1 www-data www-data        0 Feb 20 14:07 ./themes/bootstrap_darkroom/node_modules/typeface-roboto/files/.npmignore
  1185306      0 -rw-r--r--   1 www-data www-data        0 Feb 20 14:07 ./themes/bootstrap_darkroom/node_modules/typeface-pt-sans/.npmignore
  1185296      0 -rw-r--r--   1 www-data www-data        0 Feb 20 14:07 ./themes/bootstrap_darkroom/node_modules/typeface-pt-sans/files/.npmignore
  1185294      0 -rw-r--r--   1 www-data www-data        0 Feb 20 14:07 ./themes/bootstrap_darkroom/node_modules/typeface-pt-sans/files/pt-sans-latin-700.eot.mtd
  1185300      0 -rw-r--r--   1 www-data www-data        0 Feb 20 14:07 ./themes/bootstrap_darkroom/node_modules/typeface-pt-sans/files/pt-sans-latin-400.eot.mtd
  1185297      0 -rw-r--r--   1 www-data www-data        0 Feb 20 14:07 ./themes/bootstrap_darkroom/node_modules/typeface-pt-sans/files/pt-sans-latin-400.svg.mtd
  1184989      0 -rw-r--r--   1 www-data www-data        0 Feb 20 14:07 ./themes/bootstrap_darkroom/scss/custom/.keep
  1185468      0 -rw-r--r--   1 www-data www-data        0 Feb 20 14:07 ./themes/bootstrap_darkroom/css/custom/.keep
  1185084      0 -rw-r--r--   1 www-data www-data        0 Feb 20 14:07 ./themes/bootstrap_darkroom/template/menu_templates/menubar_randompic.tpl
  1435319      0 -rw-r--r--   1 www-data www-data        0 Jun 30  2022 ./include/smarty/expectException
  1311957      0 -rwxr-xr-x   1 www-data www-data        0 Oct 21 15:13 ./admin/themes/clear/css/components/general.css
  1177265      0 -rwxr-xr-x   1 www-data www-data        0 Feb 24 10:54 ./admin/user_list_backend.php
  1437642      0 -rw-rw-rw-   1 www-data www-data        0 Nov 11 17:30 ./_data/templates_c/wrt636e789b9a5669_14097981
  1181789      0 -rw-r--r--   1 www-data www-data        0 Nov 11 13:51 ./_data/i/galleries/2022/2022_02_12/P1070299-sq.JPG
  1181788      0 -rw-r--r--   1 www-data www-data        0 Nov 11 13:51 ./_data/i/galleries/2022/2022_02_12/P1070300-xx.JPG
  1177202      0 -rwxr-xr-x   1 www-data www-data        0 Oct 21 15:13 ./_data/dummy.txt
  1185637      0 -rw-r--r--   1 www-data www-data        0 Dec 20 16:22 ./local/bootstrap_darkroom/custom.css

```
Thank you.

**Comment by plegall** [comment]
> Why the file is not deleted? What is the reason?

On Piwigo 14 the file will be deleted. On Piwigo 13 we can't delete it because the minor upgrade system can no longer delete files. That's a security limit we have recently added, to avoid unwanted deletions. See https://piwigo.org/forum/viewtopic.php?pid=185219#p185219 for in-depth details.

As I couldn't have this file deleted, I made it empty. It fixes the problem. The file will be deleted when upgrading to Piwigo 14.

> In fact I found a lot of files with zero size:

and it doesn't mean it's a problem :-) It makes perfect sense to me.
